src/references/polygon-allowlist.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
can we configure this remotely?

Reply:
further iterations will handle this

Reply:
ive got a separate PR with logic for this(pared down version of rainbow token list logic) but it requires us to do some more data collection, ideally we move this all to the backend